src/connectors/index.ts

```diff
@@ -11,11 +11,14 @@ import { ChainId } from '@fuseio/fuse-swap-sdk'
 const NETWORK_URL = process.env.REACT_APP_NETWORK_URL
 const ROPSTEN_NETWORK_URL = process.env.REACT_APP_ROPSTEN_NETWORK_URL
 const MAINNET_NETWORK_URL = process.env.REACT_APP_MAINNET_NETWORK_URL
+const BINANCE_NETWORK_URL = process.env.REACT_APP_BINANCE_NETWORK_URL
 const FORMATIC_KEY = process.env.REACT_APP_FORTMATIC_KEY
 const PORTIS_ID = process.env.REACT_APP_PORTIS_ID
 
 export const NETWORK_CHAIN_ID: number = parseInt(process.env.REACT_APP_CHAIN_ID ?? '1')
 
+export const BINANCE_CHAIN_ID = 97
```

Comment:
That' the binance testnet.
In general we have two settings in each we run the dapp - testnets and mainets.
Every network (but Fuse ><) got main an and testnet.
So I suggest to have a prod config working with main networks and QA config with tesnets. 
That means the changes will be like:
- BINANCE_CHAIN_ID should be taken from env
-  ROPSTEN_NETWORK_URL/MAINNET_NETWORK_URL will be replaces with ETHEREUM_NETWORK_URL, same for ChainID.
- need to add some visible indication for testnet enc
WDYT?

Reply:
i will add a separate config so its easier to test